types/index.ts

```diff
@@ -1,3 +1,6 @@
+import { SubaccountBalanceWithToken } from '@injectivelabs/ui-common'
```

Comment:
Move this to `app/services/account/types.ts` and then export all of the types in `app/services/index.ts`